clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -163,7 +163,7 @@ public int read(byte[] b, int off, int len) throws IOException {
         try {
           mCurrentCacheStream.write(b, currentOffset, bytesRead);
         } catch (IOException ioe) {
-          LOG.warn("Failed to write into TachyonStorage, the block {} will not be in "
+          LOG.warn("Failed to write into TachyonStorage, the blockId {} will not be in "
```

Comment:
This message means the same thing as the one above - could you refactor it into a constant for the class?